metals/src/main/scala/scala/meta/internal/metals/MetalsExperimental.scala

```diff
@@ -5,7 +5,7 @@ package scala.meta.internal.metals
  * capabilities" in the initialize response.
  */
 
-object MetalsExperimental {
-  // A workaround for https://github.com/microsoft/language-server-protocol/issues/377 until it is resolved
-  val rangeHoverProvider: java.lang.Boolean = true
-}
+case class MetalsExperimental(
+    // A workaround for https://github.com/microsoft/language-server-protocol/issues/377 until it is resolved
+    val rangeHoverProvider: java.lang.Boolean = true
```

Comment:
```suggestion
    val rangeHoverProvider: Boolean = true
```

Reply:
Interesting, I always though gson didn't manage to make it work properly with scala Boolean. Or is it just an alias?

Reply:
Seems to work ok, thanks!